mungegithub/mungers/cherrypick-label-unapproved.go

```diff
@@ -69,6 +69,11 @@ func (LabelUnapprovedPicks) Munge(obj *github.MungeObject) {
 		return
 	}
 
+	if _, err := obj.GetPR(); err != nil {
+		glog.Errorf("Failed to retrieve PR: %s", err)
```

Comment:
Agreed. `IsForBranch` should have the proper error return/handling.
Hopefully the issue won't reproduce before we merge